### PR TITLE
Catchup tmo-main to v3.4.0 rc2

### PR DESCRIPTION
**Pull request by johnlange2**

### CODE REVIEW

**Comment by ryanhagen-tmo** [comment]
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| hal_espressif | https://github.com/zephyrproject-rtos/hal_espressif/commit/8e5da13712012d2cc480f7abc699235352153e05 | https://github.com/zephyrproject-rtos/hal_espressif/commit/abe299333411cb37a1cb1dd0aa2ea35c27382604 | [zephyrproject-rtos/hal_espressif@8e5da137..abe29933](https://github.com/zephyrproject-rtos/hal_espressif/compare/8e5da13712012d2cc480f7abc699235352153e05..abe299333411cb37a1cb1dd0aa2ea35c27382604) |
| mcuboot | https://github.com/zephyrproject-rtos/mcuboot/commit/6902abba270c0fbcbe8ee3bb56fe39bc9acc2774 | https://github.com/zephyrproject-rtos/mcuboot/commit/a6aef32619455cb1697c42d66db489c1608f8410 | [zephyrproject-rtos/mcuboot@6902abba..a6aef326](https://github.com/zephyrproject-rtos/mcuboot/compare/6902abba270c0fbcbe8ee3bb56fe39bc9acc2774..a6aef32619455cb1697c42d66db489c1608f8410) |
| picolibc | https://github.com/zephyrproject-rtos/picolibc/commit/93b5d5f2ad44867b60267417cd6d6250dbf68983 | https://github.com/zephyrproject-rtos/picolibc/commit/d07c38ff051386f8e09a143ea0a6c1d6d66dd1d8 | [zephyrproject-rtos/picolibc@93b5d5f2..d07c38ff](https://github.com/zephyrproject-rtos/picolibc/compare/93b5d5f2ad44867b60267417cd6d6250dbf68983..d07c38ff051386f8e09a143ea0a6c1d6d66dd1d8) |
| zscilib | https://github.com/zephyrproject-rtos/zscilib/commit/0035be5e6a45e4ab89755b176d305d7a877fc79c | https://github.com/zephyrproject-rtos/zscilib/commit/34c3432e81085bb717e4871d21ca419ae0058ec5 (master) | [zephyrproject-rtos/zscilib@0035be5e..34c3432e](https://github.com/zephyrproject-rtos/zscilib/compare/0035be5e6a45e4ab89755b176d305d7a877fc79c..34c3432e81085bb717e4871d21ca419ae0058ec5) |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

**Comment by DRuffer-tmo** [comment]
John, why are you duplicating what I am working on?

**Comment by DRuffer-tmo** [comment]
Please don't merge this until I have had a chance to review it in regard to the work that I've already been working on.

File: .gitattributes [hunk 1]
```diff
@@ -62,14 +62,19 @@ boards/arm/efr32mg_sltb004a/efr32mg_sltb004a-pinctrl.dtsi merge=ours
 boards/arm/efr32mg_sltb004a/efr32mg_sltb004a.dts merge=ours
 boards/arm/efr32mg_sltb004a/efr32mg_sltb004a_defconfig merge=ours
 boards/arm/gd32a503v_eval/gd32f450z_eval.yaml merge=ours
+boards/arm/tmo_dev_edge/CMakeLists.txt merge=ours
+boards/arm/tmo_dev_edge/Kconfig.board merge=ours
 boards/arm/tmo_dev_edge/Kconfig.defconfig merge=ours
 boards/arm/tmo_dev_edge/board.c merge=ours
+boards/arm/tmo_dev_edge/board.cmake merge=ours
 boards/arm/tmo_dev_edge/board.h merge=ours
 boards/arm/tmo_dev_edge/doc/index.rst merge=ours
```
[the remark this answers is not in the document]
Comment (johnlange2) [reply .gitattributes:71]: right, it needs to be added to .gitattributes

**Pull request closed without merging**